content/chainguard/developer-resources.md

```diff
@@ -0,0 +1,218 @@
+---
+title: "Developer Resources"
+lead: "AI-ready documentation bundle for development"
+description: "Compiled Chainguard documentation optimized for use with AI coding assistants"
+type: "article"
+date: 2025-07-29T10:00:00+00:00
+lastmod: 2025-07-29T10:00:00+00:00
+draft: false
+images: []
+weight: 50
+toc: false
+menu:
+  main:
+    weight: -1
```

Comment:
```suggestion
    weight:  9998
```

I think will put these docs next to each other in the sidebar

Reply:
I want it only on the bottom part of the sidebar, so I removed it